x/upgrade/keeper/keeper_test.go

```diff
@@ -278,6 +278,40 @@ func (s *KeeperTestSuite) TestLastCompletedUpgrade() {
 	require.Equal(int64(15), height)
 }
 
+func (s *KeeperTestSuite) TestLastCompletedUpgradeOrdering() {
```

Comment:
```suggestion
// This test ensures that `GetLastDoneUpgrade` always returns the last upgrade according to the block height
// it was executed at, rather than using an ordering based on upgrade names.
func (s *KeeperTestSuite) TestLastCompletedUpgradeOrdering() {
```